dist/stat_data.py

```diff
@@ -820,19 +820,26 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Compression statistics
     ##########################################
-    CompressStat('compress_hist_ratio_2', 'number of blocks with compress ratio smaller than 2'),
-    CompressStat('compress_hist_ratio_4', 'number of blocks with compress ratio smaller than 4'),
-    CompressStat('compress_hist_ratio_8', 'number of blocks with compress ratio smaller than 8'),
-    CompressStat('compress_hist_ratio_16', 'number of blocks with compress ratio smaller than 16'),
-    CompressStat('compress_hist_ratio_32', 'number of blocks with compress ratio smaller than 32'),
-    CompressStat('compress_hist_ratio_64', 'number of blocks with compress ratio smaller than 64'),
-    CompressStat('compress_hist_ratio_max', 'number of blocks with compress ratio greater than 64'),
     CompressStat('compress_precomp_intl_max_page_size', 'compressed page maximum internal page size prior to compression', 'no_clear,no_scale,size'),
     CompressStat('compress_precomp_leaf_max_page_size', 'compressed page maximum leaf page size prior to compression ', 'no_clear,no_scale,size'),
-    CompressStat('compress_read', 'compressed pages read'),
-    CompressStat('compress_write', 'compressed pages written'),
-    CompressStat('compress_write_fail', 'page written failed to compress'),
-    CompressStat('compress_write_too_small', 'page written was too small to compress'),
+    CompressStat('compress_read', 'compressed blocks read'),
+    CompressStat('compress_read_ratio_hist_2', 'compressed blocks read with compression ratio smaller than  2'),
```

Comment:
Small stylistic note. Since stats or sorted alphabetically on their descriptions I've slipped in two spaces before the single digits ranges (2,4,8). This means the stats are sorted 2,4,8,16,32,64 instead of 16,2,32,4,64,8

Reply:
I wonder if this deserves a comment in the file so no one treats it as a mistake.

Reply:
SGTM 👍. Updated in [e544af4](https://github.com/wiredtiger/wiredtiger/pull/9908/commits/e544af4b96440d7ff122e6a40fb7fcf85334bf0c)